src/main/java/Opus.java

```diff
@@ -0,0 +1,70 @@
+import java.util.Scanner;
+
+public class Opus {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println(" Hello! I'm Opus");
+        System.out.println(" What can I do for you?");
+
+        Task[] tasks = new Task[101];
+        int taskCount = 0;
+
+        while(true) {
+            String s = scanner.nextLine();
+            String[] words = s.split(" ");
+
+            try{
+                if (s.equals("bye")) {
+                    break;
+                } else if (s.equals("list")) {
+                    for (int i = 1; i <= taskCount; i++) {
+                        System.out.println(i + ". " + tasks[i]);
+                    }
+                } 
+                else if (words[0].equals("mark")) {
+                    int i = Integer.parseInt(words[1]);
+                    tasks[i].markAsDone();
+                    System.out.println("Nice! I've marked this task as done:");
+                    System.out.println(tasks[i]);
+                } 
+                else if (words[0].equals("delete")) {
+                    int i = Integer.parseInt(words[1]);
+                    System.out.println("Noted. I've removed this task:");
+                    System.out.println("   " + tasks[i]);
+                    for (int j = i; j < taskCount; j++) {
+                        tasks[j] = tasks[j + 1]; 
+                    }
+                    taskCount--; 
+                    System.out.println(" Now you have " + taskCount + " tasks in the list.");
```

Comment:
Would it be better to extract out the tasks into individual classes? Perhaps it will make the code look cleaner